### PR TITLE
Add pocket 404 and 500 pages (fixes #11542)

### LINKED ISSUE #11542: Add 404 and 500 Pages to Pocket mode
This ticket is blocked until https://github.com/mozilla/bedrock/pull/11494 is resolved and merged

## Description

If you run Bedrock in its new Pocket mode locally (`npm run in-pocket-mode` or `make run-pocket`) and you go to `/404/` or `/500/` there is no page available - previously we were going to use the Bedrock 404 and 500 pages, i'm assuming, but we should take the opportunity to bring over the ones that Pocket use

---

#### Success Criteria
- [ ] /404/ in Pocket mode serves a 404 page suitable for Pocket, locally, from bedrock
- [ ] /500/ in Pocket mode serves a 500 page suitable for Pocket, locally, from bedrock
- [ ] Both pages are marked up with Fluent strings

### PR DESCRIPTION
**Pull request by maureenlholland**
## One-line summary

Adds Pocket 500/404 translations and styling. The pages should be cleanly styled and easy to read, showing appropriate template depending on the error (500 = server error, 404 = page not found).

Note: the 404 is using existing Pocket copy but not the existing styles: https://getpocket.com/en/asg

⚠️ we might never see the 404 in production, but it doesn't hurt to have a backup

## Issue / Bugzilla link
https://github.com/mozilla/bedrock/issues/11542

## Testing

To test this work:

http://localhost:8000/en/404/
http://localhost:8000/en/500/

### CODE REVIEW

File: bedrock/pocket/templates/pocket/includes/platform-footer.html
Comment (maureenlholland) [inline bedrock/pocket/templates/pocket/includes/platform-footer.html:29]: I added these because it's helpful to have HTML size defaults in case CSS does not load or is blocked by user

File: bedrock/pocket/templates/pocket/404.html
Comment (reemhamz) [reply bedrock/pocket/templates/pocket/404.html:19]: Nope, it shouldn't. Whenever I'm logged-in in Pocket and clicked the homepage URL, it always directs me to https://getpocket.com/my-list (aka my account's saved items). It's no different here

File: bedrock/pocket/templates/pocket/includes/platform-footer.html
Comment (reemhamz) [reply bedrock/pocket/templates/pocket/includes/platform-footer.html:29]: smart!

File: media/css/pocket/includes/_platform-footer.scss
Comment (reemhamz) [inline media/css/pocket/includes/_platform-footer.scss:109]: **Praise:** oops, thanks for catching this!

File: media/css/pocket/components/error-page.scss
Comment (reemhamz) [inline media/css/pocket/components/error-page.scss:8]: **Suggestion (non-blocking):** Typically in Pocket and the rest of our redesign, most of our inline anchor tags are either:
- colored grey with teal in pseudo states (i.e. https://dev.tekcopteg.com/en/contact-info/) OR
- colored teal with grey in pseudo states (i.e. https://dev.tekcopteg.com/en/about/)

I think it would be nice and consistent to stick to the color scheme. I see in the [original 404 page](https://getpocket.com/en/404) it's teal (with no color change in pseudo state). How about we make the anchor tags colored teal with the grey hover states?

File: bedrock/pocket/views.py
Comment (reemhamz) [inline bedrock/pocket/views.py:10]: **Question (non-blocking):** Should we also add these templates to the `pocket/urls.py` file?

**Review by reemhamz — APPROVED** [review]
Super noice 🧚🏼✨! 
Just a couple of suggestions, but otherwise r+wc 🧠

File: bedrock/pocket/templates/pocket/404.html
Comment (maureenlholland) [reply bedrock/pocket/templates/pocket/404.html:19]: Cool - thanks for checking, will switch to pocket.home

File: media/css/pocket/components/error-page.scss
Comment (maureenlholland) [reply media/css/pocket/components/error-page.scss:8]: Good catch! Yes will line up with with about styles

File: bedrock/pocket/views.py
Comment (maureenlholland) [reply bedrock/pocket/views.py:10]: Based on the way Bedrock's 404/500 templates are handled, this is the only place they need to be set.

**Comment by maureenlholland** [comment]
Thanks for the review @reemhamz ! Couple of suggested changes made, if you can take a quick look before merge, that would be 💯

**Comment by reemhamz** [comment]
Super neat! r+ 🍇

**Pull request merged**
